src/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -318,6 +325,12 @@ public GenotypeLikelihoodsAllelePair(final int alleleIndex1, final int alleleInd
         }
     }
 
+    /**
+     * Calculate the cache of diploid alleles for each PL index
+     *
+     * @param altAlleles   number of alternate alleles
+     * @return cache of diploid alleles for each PL index
+     */
     private static GenotypeLikelihoodsAllelePair[] calculatePLcache(final int altAlleles) {
```

Comment:
might as well rename this function (its private anyhow) to calculateDiploidPLcache